react-time-track/src/views/home.js

```diff
@@ -53,14 +55,18 @@ function Home({ children }) {
           gridArea: "header"
         }}
       >
-        <Header tittle="Projects" path="/" />
+        <Header
+          tittle={currentUser.role === "Analyst" ? "My Status" : "Projects"}
```

Comment:
Just an observation, please, don't do it now as it is not a bug.
It is written as `title`.

Reply:
Ok, thanks